portal-web/test/functional/com/liferay/portalweb/tests/enduser/adaptivemedia/AdaptiveMedia.testcase

```diff
@@ -452,6 +511,46 @@
 		</execute>
 	</command>
 
+	<command name="EditImageResolutionWithNoDimensions" priority="4">
+		<var name="key_imageName" value="testResolution" />
```

Comment:
Where is this being used in the test? Generally you shouldn't be declared key variables at the testcase level.

Reply:
I'm going to remove this line. This test passed without it.